prow/cmd/pipeline/main.go

```diff
@@ -117,7 +117,10 @@ func main() {
 		logrus.WithError(err).Fatal("failed to load prow config")
 	}
 
-	configs, err := kube.LoadClusterConfigs(kube.NewConfig(kube.ConfigFile(o.kubeconfig)))
+	configs, err := o.kubernetes.LoadClusterConfigs(func() {
+		logrus.Info("Kubeconfig changed, exiting to trigger a restart")
```

Comment:
we could just `Fatal` here, right?

Reply:
I mean that at this point the context is unused. See https://github.com/kubernetes/test-infra/pull/28990/files#diff-37ab81c8ea5f708e7ed05d8b8bafd95dcca10399dbdc411e37d0a50481dae780R149

Reply:
yeah, you are right, let me change that to fatal